twill-yarn/src/main/java/org/apache/twill/yarn/YarnTwillPreparer.java

```diff
@@ -151,8 +152,9 @@
     this.reservedMemory = yarnConfig.getInt(Configs.Keys.JAVA_RESERVED_MEMORY_MB,
                                             Configs.Defaults.JAVA_RESERVED_MEMORY_MB);
     this.extraOptions = extraOptions;
-    this.logLevel = logLevel;
     this.classAcceptor = new ClassAcceptor();
+    this.logLevel = logLevel;
```

Comment:
Shouldn't have this field anymore. Every log level info should be in the `logLevels` map